src/task_11_01.py

```diff
@@ -0,0 +1,243 @@
+""" Задание 11.1
+
+Создать пять классов описывающие реальные объекты. Каждый класс
+должен содержать минимум три приватных атрибута, конструктор, геттеры
+и сеттеры для каждого атрибута, два метода."""
+
+
+class Wine:
+    """Class of wine instances
+
+    Describes the wine instance based on properties:
+    year - the year of harvest
+    color - color of the wine
+    grape_sort - sort of the grape
+    is_sparkling - tag of sparkling feature
+    """
+
+    COLORS = {"Red", "White", "Rose"}
+
+    def __init__(self, year, color="Red", grape_sort="Merlot", is_sparkling=False):
+        self.year = year
+        self.color = color
+        self.grape_sort = grape_sort
+        self.is_sparkling = is_sparkling
+
+    @property
+    def color(self) -> str:
+        return self.__color
+
+    @color.setter
+    def color(self, new_color: str):
+        if new_color.capitalize() in Wine.COLORS:
```

Comment:
```suggestion
        if new_color.capitalize() in self.COLORS:
```

Reply:
@tamkovich 
This is a class variable now, thats is why acces to  .COLORS is more correct throught class identifier... i guess

Reply:
@dzmitrysk Image the situation when somebody decided to reuse your code:
- Bob created class BobWine inherited from Wine
- Bob overrides COLORS variables with set of {'Purple', 'Blue'}

Actual Results:
- color setter still accepts only {"Red", "White", "Rose"}
- Bob is sad :(

Expected Results:
- color setter will work with the Bob's values
- Bob likes your class 👍

Reply:
Wow!
Explanation opens up a new universe, as opposed to a simple instruction to change something.

applied